spec/services/orangelight_document_spec.rb

```diff
@@ -0,0 +1,140 @@
+                                         technique: "Cast",
+                                         weight: "8.26")
+      end
+      let(:issue) do
+        FactoryBot.create_for_repository(:numismatic_issue,
+                                         numismatic_citation_ids: [citation.id],
+                                         member_ids: [coin.id],
+                                         object_type: "coin",
+                                         date_range: date_range,
+                                         denomination: "1/2 Penny",
+                                         metal: "copper",
+                                         shape: "round",
+                                         color: "green",
+                                         edge: "GOTT MIT UNS",
+                                         era: "uncertain",
+                                         ruler: "George I",
+                                         master: "William Wood",
+                                         workshop: "Bristol",
+                                         series: "Hibernia",
+                                         place: "Great Britain",
+                                         obverse_figure: "bust",
+                                         obverse_symbol: "cornucopia",
+                                         obverse_part: "standing",
+                                         obverse_orientation: "right",
+                                         obverse_figure_description: "Harp at left side, 5 strings.",
+                                         obverse_figure_relationship: "Victory behind",
+                                         obverse_legend: "GEORGIUS•DEI•GRATIA•REX•",
+                                         obverse_attributes: ["to left and right", "around edge"],
+                                         reverse_figure: "Hibernia",
+                                         reverse_symbol: "goat head",
+                                         reverse_part: "seated",
+                                         reverse_orientation: "left",
+                                         reverse_figure_description: "Harp at right side, 11 strings. Right arm holding up a palm-branch",
+                                         reverse_figure_relationship: "corn-ear behind head",
+                                         reverse_legend: "•HIBERNIA•1723•",
+                                         reverse_attributes: ["above", "2 within Є"],
+                                         artist: ["André Galle"],
+                                         subject: ["unicorn"])
+      end
+
+      before do
+        stub_ezid(shoulder: "99999/fk4", blade: "123456")
+        coin
+        issue
+      end
+
+      it "returns a IIIF manifest for a resource with a file" do
+        output = MultiJson.load(builder.to_json, symbolize_keys: true)
+        expect(output[:id]).to eq "coin-1"
```

Comment:
Same question as above: any reason not to test for `"coin-#{coin.coin_number}"` ?